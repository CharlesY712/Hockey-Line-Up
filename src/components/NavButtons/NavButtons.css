img {
  width: 80%;
  height: 40%;
}

.nav-buttons {
  position: fixed;
  left: 0;
  bottom: 0;
  width: 100%;
  background-color: rgb(45, 45, 45);
  color: #a3a8ae;
  text-align: center;
  display: flex;
  justify-content: space-evenly;
}

.button {
  box-sizing: border-box;
  height: 25vw;
  margin: auto auto;
  padding-top: 4%; 
  text-decoration: none;
  font-size: 4.5vw;
  color: #A3A8AE;
  font-family: 'Allerta Stencil', sans-serif;
  border: .1rem outset #4E5357;
}

a.active {
  background-color:rgb(40, 80, 105)
}

@media (min-width: 768px) {
  .icon {
    width: 20%;
    height: 25px;
    margin-right: 0vw;
  }

  .nav-buttons {
    position: fixed;
    left: 0;
    bottom: 81.1vh;
    width: 100%;
    text-align: center;
  }
  
  .button {
    height: 45px;
    padding-top: 0; 
    font-size: 2rem;
  }
}

@media (min-width: 912px) {

}

@media (min-width: 1056px) {
}

@media (min-width: 1200px) {
}