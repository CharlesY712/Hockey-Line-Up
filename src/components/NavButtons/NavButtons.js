import React from 'react';
import { NavLink } from 'react-router-dom';
import './NavButtons.css';
import * as icon from '../../images/icons/';

const NavButtons = () => {

  return (
    <div className="nav-buttons">
      <NavLink to="/" className="home button">
        <img alt="home-icon" src={icon.home}/>Home
      </NavLink>    
      <NavLink to="/day" className="day button">
        <img alt="day-icon" src={icon.day}/>Day
      </NavLink>
      <NavLink to="/week" className="week button">
        <img alt="week-icon" src={icon.week}/>Week
      </NavLink>
      <NavLink to="/month" className="month button">
        <img alt="month-icon" src={icon.month}/>Month
      </NavLink>
    </div>
  );
};

export default (NavButtons);
