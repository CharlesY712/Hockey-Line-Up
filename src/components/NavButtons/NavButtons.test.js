import React from 'react';
import NavButtons from './NavButtons';
import { shallow } from 'enzyme';

describe('App', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<NavButtons/>);
  });

  it('matches the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
