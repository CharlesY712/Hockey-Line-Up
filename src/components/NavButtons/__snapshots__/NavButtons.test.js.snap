// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NavButtons />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          className="home button"
          exact={true}
          to="/"
>
          <img
                    alt="home-icon"
                    className="icon"
                    src="home.svg"
          />
          Home
</NavLink>,
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          className="day button"
          to="/day"
>
          <img
                    alt="day-icon"
                    className="icon"
                    src="day-calendar.svg"
          />
          Day
</NavLink>,
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          className="week button"
          to="/week"
>
          <img
                    alt="week-icon"
                    className="icon"
                    src="week-calendar.svg"
          />
          Week
</NavLink>,
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          className="month button"
          to="/month"
>
          <img
                    alt="month-icon"
                    className="icon"
                    src="month-calendar.svg"
          />
          Month
</NavLink>,
      ],
      "className": "nav-buttons",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "ariaCurrent": "true",
          "children": Array [
            <img
              alt="home-icon"
              className="icon"
              src="home.svg"
/>,
            "Home",
          ],
          "className": "home button",
          "exact": true,
          "to": "/",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "home-icon",
              "className": "icon",
              "src": "home.svg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          "Home",
        ],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "ariaCurrent": "true",
          "children": Array [
            <img
              alt="day-icon"
              className="icon"
              src="day-calendar.svg"
/>,
            "Day",
          ],
          "className": "day button",
          "to": "/day",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "day-icon",
              "className": "icon",
              "src": "day-calendar.svg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          "Day",
        ],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "ariaCurrent": "true",
          "children": Array [
            <img
              alt="week-icon"
              className="icon"
              src="week-calendar.svg"
/>,
            "Week",
          ],
          "className": "week button",
          "to": "/week",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "week-icon",
              "className": "icon",
              "src": "week-calendar.svg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          "Week",
        ],
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "ariaCurrent": "true",
          "children": Array [
            <img
              alt="month-icon"
              className="icon"
              src="month-calendar.svg"
/>,
            "Month",
          ],
          "className": "month button",
          "to": "/month",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "month-icon",
              "className": "icon",
              "src": "month-calendar.svg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          "Month",
        ],
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            className="home button"
            exact={true}
            to="/"
>
            <img
                        alt="home-icon"
                        className="icon"
                        src="home.svg"
            />
            Home
</NavLink>,
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            className="day button"
            to="/day"
>
            <img
                        alt="day-icon"
                        className="icon"
                        src="day-calendar.svg"
            />
            Day
</NavLink>,
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            className="week button"
            to="/week"
>
            <img
                        alt="week-icon"
                        className="icon"
                        src="week-calendar.svg"
            />
            Week
</NavLink>,
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            className="month button"
            to="/month"
>
            <img
                        alt="month-icon"
                        className="icon"
                        src="month-calendar.svg"
            />
            Month
</NavLink>,
        ],
        "className": "nav-buttons",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "ariaCurrent": "true",
            "children": Array [
              <img
                alt="home-icon"
                className="icon"
                src="home.svg"
/>,
              "Home",
            ],
            "className": "home button",
            "exact": true,
            "to": "/",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "home-icon",
                "className": "icon",
                "src": "home.svg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            "Home",
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "ariaCurrent": "true",
            "children": Array [
              <img
                alt="day-icon"
                className="icon"
                src="day-calendar.svg"
/>,
              "Day",
            ],
            "className": "day button",
            "to": "/day",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "day-icon",
                "className": "icon",
                "src": "day-calendar.svg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            "Day",
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "ariaCurrent": "true",
            "children": Array [
              <img
                alt="week-icon"
                className="icon"
                src="week-calendar.svg"
/>,
              "Week",
            ],
            "className": "week button",
            "to": "/week",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "week-icon",
                "className": "icon",
                "src": "week-calendar.svg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            "Week",
          ],
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "ariaCurrent": "true",
            "children": Array [
              <img
                alt="month-icon"
                className="icon"
                src="month-calendar.svg"
/>,
              "Month",
            ],
            "className": "month button",
            "to": "/month",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "month-icon",
                "className": "icon",
                "src": "month-calendar.svg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            "Month",
          ],
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
